attestation-agent/attester/Cargo.toml

```diff
@@ -18,15 +18,17 @@ serde_json.workspace = true
 sev = { git = "https://github.com/virtee/sev", version = "1.2", default-features = false, features = ["snp"], optional = true }
 strum.workspace = true
 tdx-attest-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", tag = "DCAP_1.16", optional = true }
+nix = {version = "0.26.2", optional = true }
```

Comment:
Could you put this dependency to follow the dictionarial order?

Reply:
Done